### PR TITLE
GDEV-105: Use the chassis lib to parse and use config

### PR DESCRIPTION
**Pull request by deepakunni3**
Closes GDEV-105

### CODE REVIEW

File: metadata_service/__main__.py [hunk 1]
```diff
@@ -18,13 +18,15 @@
...
+from ghga_service_chassis_lib.api import run_server
+from metadata_service.config import get_config
 from metadata_service.api import app
 
 
 def run(config: Optional[str] = typer.Option(None, help="Path to config yaml.")):
     """Starts backend server"""
-    # TODO: implement config
-    uvicorn.run(app)
+    #uvicorn.run(app)
```
Comment (MoritzHahn1337) [inline metadata_service/__main__.py:28]: As far as I remember, we agreed on not keeping legacy code in comments

File: tests/integration/test_dataset_routes.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import pytest
+from tests.fixtures import initialize_test_db, api_client
+
+
+@pytest.mark.skip()
```
Comment (MoritzHahn1337) [inline tests/integration/test_dataset_routes.py:5]: If I understand pytest.mark.skip() correctly, this skips the test alltogether. Is this the intended behaviour?

File: metadata_service/__main__.py (hunk 1 above)
Comment (deepakunni3) [reply metadata_service/__main__.py]: Thanks @MoritzHahn1337 ! Legacy code removed.

File: tests/integration/test_dataset_routes.py (hunk 2 above)
Comment (deepakunni3) [reply tests/integration/test_dataset_routes.py]: Thanks for catching that! 

Fixed the tests in the latest commits.

**Review by ac-jorellanaf — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@ FROM python:3.9.6-buster
 
 COPY . /service
 WORKDIR /service
-RUN pip install -r requirements.txt
+RUN python setup.py install
```
Comment (KerstenBreuer) [inline Dockerfile:20]:
```suggestion
RUN pip install .
```
Sorry @deepakunni3, this was wrong in the template repository but is now fixed.

**Review by KerstenBreuer — APPROVED** [review]
Perfect

File: Dockerfile (hunk 3 above)
Comment (deepakunni3) [reply Dockerfile:20]: No worries! thanks for the pointer 👍

**Pull request merged**